candi/bashible/bundles/centos/bootstrap.sh.tpl

```diff
@@ -40,6 +40,13 @@ if [ "${VERSION_ID}" == "8" ] ; then
   alternatives --set python /usr/bin/python3
 fi
 
+for FS_NAME in `egrep -v -e '(^[[:space:]]*$|^#)' /etc/fstab | awk '{ print $1 }'`; do
```

Comment:
Why we use /etc/fstab instead of mount | grep xfs ?

Reply:
Didn't thought about mount in this case. Fixed.